site/docs/v1/tech/apis/scim/SCIMServiceProvider.adoc

```diff
@@ -20,18 +19,32 @@ This API is used to retrieve information about the configuration of the link:htt
 
 === Request
 [.api-authentication]
-link:/docs/v1/tech/apis/authentication#no-authentication-required[icon:unlock[type=fas]] Retrieve All Resource Types
+link:/docs/v1/tech/apis/authentication#client-credentials[icon:cogs[type=fas]] Retrieve All Resource Types
```

Comment:
This only retrieves all resources if you omit the parameter. Review similar APIs for how we document w/ and w/out a constraint. For example Retrieve All Applications or Retrieve by Id.